### DISCUSSION TITLE
How do I tell vcpkg to build a specific feature of a submodule (f.e. in case of Qt 5)?

### DISCUSSION THREAD

**Discussion by i-C-o-d-e-r** [microsoft/vcpkg#26296] [Q&A] [answered]
Hi there,

I would like to use Qt 5 and therefore would like to use the port qt5. This is obviously a convenience package, which acts as wrapper for all the different submodules of Qt 5. But the port of the submodule qt5-base supports the feature `postgresqlplugin`, which is only build if explicitly requested like this `vcpkg install qt5-base[postgresqlplugin]`.

But as I use the wrapper port, I use `vcpkg install qt5` instead. So what is the proper syntax then?

A clarification is highly appreciated.

**Comment by Osyotr** [comment, marked as answer]
`vcpkg install qt5 qt5-base[postgresqlpugin]`
But I don't recommend using `qt5` metaport as it's somewhat unmaintained. Use `qt5-*` ports instead.

**Reply by i-C-o-d-e-r** [reply]
@Osyotr Thank you! That worked for me.
